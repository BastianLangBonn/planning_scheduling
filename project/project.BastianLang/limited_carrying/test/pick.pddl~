(define (problem problem1)
	(:domain complex_domain)
	(:objects 	s - s1
			o - o1
			r - robot)
	(:init 	(at_location r s)
		(at_location o s)
		(gripper_free r)	

		;Distance Metric

		(= (distance s s) 0)
		
		(= (total-cost) 0)
		
	)
	(:goal
		(and (not(gripper_free r))(grapped r o)(not(at_location o s)))
	)
)
