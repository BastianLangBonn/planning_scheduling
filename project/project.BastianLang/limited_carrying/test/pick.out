(pick o2 s)
(stage o2)
(pick o1 s)
(stage o1)
(pick o3 s)
